edgelet/iotedge/src/check/checks/hostname.rs

```diff
@@ -86,8 +86,12 @@ impl Hostname {
         //
         // Instead, we punt on this check and assume that everything's fine if config_hostname is identical to the device hostname,
         // or starts with it.
+        //
+        // Azure FQDN don't support capital letter so we lower case the hostname before doing the check.
         if config_hostname != machine_hostname
```

Comment:
to_lowercase() these as well.

Reply:
Ok, though that does't change the logic.